openedx/core/lib/edx_api_utils.py

```diff
@@ -19,12 +19,7 @@ def get_fields(fields, response):
     """Extracts desired fields from the API response"""
     results = {}
     for field in fields:
-        try:
-            results[field] = response[field]
-        # TODO: Determine what exception would be raised here if response does not have the specified field
-        except:
-            msg = '{resource} does not have the attribute {field}'.format(resource, field)
-            log.exception(msg)
+        results[field] = response.get(field)
```

Comment:
Do we no longer care about logging the request for a non-existent field?

Reply:
I had originally introduced that logging in my most recent ticket without being prompted to, and when working on this ticket noticed that an exception isn't raised in that situation anyway, but I can add an if statement if you think that the log message would be helpful / important.

Reply:
Gotcha, if this is all new and you don't think it's necessary, then feel free to remove it.  I just didn't want to rip it out if anyone had come to use these warnings in NewRelic or Splunk for anything.